midp/background.js

```diff
@@ -54,7 +54,19 @@ function backgroundCheck() {
   fgMidletNumber = (bgServer == 2) ? 1 : 2;
   fgMidletClass = MIDP.manifest["MIDlet-" + fgMidletNumber].split(",")[2];
 
-  DumbPipe.close(DumbPipe.open("backgroundCheck", {}));
+  if (!MIDlet.shouldStartBackgroundService ||
```

Comment:
Why `true` by default? Messaging midlets like WhatsApp seem like the exception rather than the rule, so my inclination would be to set it to `false` by default, so other midlets don't start in the background unless we explicitly enable them to do so.